lib/manifest/PlaylistGenerator.js

```diff
@@ -78,15 +77,16 @@ function savePlaylist() {
 
 PlaylistGenerator.prototype.updatePlaylist = function() {
     var that = this;
-    var currTime = new Date();
     // Update the entry's playlist every minute
-    if ((currTime - that.lastPlaylistUpdate) > config.get('playlistUpdateInterval')) {
-        that.logger.info("Updating playlist");
-        return savePlaylist.call(this)
-            .catch(function(error) {
-                that.logger.error("Unable to update playlist: %s", error);
-            });
-    }
+    that.logger.debug("Retrieving flavors for update");
```

Comment:
question: what happens if you call updatePlaylist, then the stream stops, and then the getAllFlavours return?